packages/react-app/src/StatCard.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+const StatCard = props => {
```

Comment:
Would be cool to also import this into the Artist.js page as I think it's now a common component